R/read10xVisium.R

```diff
@@ -125,7 +125,7 @@ read10xVisium <- function(samples="",
     
     dir <- file.path(samples, "spatial")
     xyz <- file.path(
-        rep(dir, each = length(sids)), 
+        rep(dir, each = 2), 
```

Comment:
(same comment on the RELEASE_3_16 PR): could we perhaps assign the character string in line 131 to a variable `sfx` (or something for file name suffix), and use `length(sfx)` here? I feel like the `2` isn't very explicit or robust. E.g., I added another suffix not changing the 2 and it runs silently, but samples/files get mixed up... just to keep things clear and safe for the future.

Reply:
Ah yes this could work, thanks. I'll try it.